### PR TITLE
Changelog review.

### PR DESCRIPTION
**Pull request by Natim**
r? @ametaireau

### CODE REVIEW

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Contributors
+============
+
+* Alexis Metaireau <alexis@mozilla.com>
+* Fil <fil@rezo.net>
+* Mathieu Agopian <mathieu@agopian.info>
+* Mathieu Leplatre <mathieu@mozilla.com>
+* Nicolas Perriault <nperriault@mozilla.com>
+* Rémy Hubscher <rhubscher@mozilla.com>
+* Tarek Ziade <tarek@mozilla.com>
```
Comment (ametaireau) [inline CONTRIBUTORS.rst]: Cool, we need to make sure we update this file each time we do a release (if we had templates of what we should do for a release, we could include this step)

File: docs/conf.py [hunk 2]
```diff
@@ -121,6 +121,7 @@
 texinfo_documents = [
   ('index', 'Kinto', u'Kinto Documentation',
    u'Mozilla Services — Da French Team', 'Kinto',
-   'A storage service for data sync, share with fine-grained permissions.',
+   'A storage service for data storage, synchronization and sharing '
```
Comment (ametaireau) [inline docs/conf.py]: "A storage service for storage" sounds weird to me. The fact that we have fine grained permissions is IMO too much detailed information. I would rephrase this with "A remote storage service with syncing and sharing abilities"?

File: setup.py [hunk 3]
```diff
@@ -1,14 +1,18 @@
 import os
+import codecs
 from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(here, 'README.rst')) as f:
+with codecs.open(os.path.join(here, 'README.rst'), encoding='utf-8') as f:
```
Comment (ametaireau) [inline setup.py]: we could put that in a function named `read_file`.

**Comment by ametaireau** [comment]
LGTM with comments.

File: CONTRIBUTORS.rst (hunk 1 above)
Comment (Natim) [inline CONTRIBUTORS.rst]: I have updated this: https://github.com/mozilla-services/kinto/pull/86#issue-91756118

Not sure if we put the template somewhere else.
